feature/bgp/static-route_bgp_redistribution/README.md

```diff
@@ -0,0 +1,653 @@
+*   Verify a tag-set with name ```tag-set-v4``` is configured
+    *   /routing-policy/defined-sets/tag-sets/tag-set/state/name
+*   Verify tag-set ```tag-set-v4``` with a tag value of ```100``` is configured
+    *   /routing-policy/defined-sets/tag-sets/tag-set/state/tag-value
+*   Verify for routing-policy ```route-policy-v4``` statement ```statement-v4``` tag-set is set to ```tag-set-v4```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/conditions/match-tag-set/state/tag-set
+*   Verify for routing-policy ```route-policy-v4``` statement ```statement-v4``` match-set-options is set to ```ANY```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/conditions/match-tag-set/state/match-set-options
+##### Validate test results
+*   Verify that the ATE does not receives the redistributed static route ```ipv4-route```
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/prefix
+
+### RT-1.27.13 [TODO: https://github.com/openconfig/featureprofiles/issues/2568]
+#### Redistribution of IPv4 static routes to BGP that matches a conditional tag should happen
+---
+##### Configure a tag-set with correct tag value to validate that the route is redistributed
+*   Configure tag-set ```tag-set-v4``` with a tag value of ```40```
+    *   /routing-policy/defined-sets/tag-sets/tag-set/config/tag-value
+##### Verification
+*   Verify tag-set ```tag-set-v4``` with a tag value of ```40``` is configured
+    *   /routing-policy/defined-sets/tag-sets/tag-set/state/tag-value
+##### Validate test results
+*   Verify that the ATE receives the redistributed static route ```ipv4-route```
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/prefix
+*   Initiate traffic from ATE port-1 to the DUT and destined to ```ipv4-network``` i.e. ```192.168.10.0/24```
+*   Validate that the traffic is received on ATE port-2
+
+### RT-1.27.14 [TODO: https://github.com/openconfig/featureprofiles/issues/2568]
+#### Redistribute a NULL IPv4 static routes to BGP with a next-hop configured through route-policy
+---
+##### Configure a NULL static route
+*   Configure an IPv4 static route ```ipv4-drop-route``` on DUT destined to ```ipv4-drop-network``` i.e. ```192.168.20.0/24``` with the next hop set to ```DROP```
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/config/prefix
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/next-hop
+##### Configure a tag on the static route
+*   Set a tag on the ```ipv4-drop-route``` to ```40```
+        *   /network-instances/network-instance/protocols/protocol/static-routes/static/config/set-tag
+##### Configure BGP actions to set a next-hop
+*   For routing-policy ```route-policy-v4``` statement ```statement-v4``` set next-hop to ```192.168.1.9```
```

Comment:
In DCGate usecase, we do not need to set NH for NULL static routes as thye are going to be advertised over eBGP enly, henc NH would be overwritten anyway.
But it do not harm either.